staging/cse/windows/kubeletfunc.ps1

```diff
@@ -209,6 +209,67 @@ function Get-KubePackage {
     Remove-Item $zipfile
 }
 
+function Remove-KubeletNodeLabel {
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $KubeletNodeLabels,
+        [Parameter(Mandatory=$true)][string]
+        $Label
+    )
+
+    $labelList = $KubeletNodeLabels -split ","
+    $filtered = $labelList | Where-Object { $_ -ne $Label }
+    return $filtered -join ","
+}
+
+function Get-TagValue {
+    Param(
+        [Parameter(Mandatory=$true)][string]
+        $TagName,
+        [Parameter(Mandatory=$true)][string]
+        $DefaultValue
+    )
+
+    $uri = "http://169.254.169.254/metadata/instance?api-version=2021-02-01"
+    try {
+        $response = Retry-Command -Command "Invoke-RestMethod" -Args @{Uri=$uri; Method="Get"; ContentType="application/json"; Headers=@{"Metadata"="true"}} -Retries 3 -RetryDelaySeconds 5
+    } catch {
+        Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_LOOKUP_INSTANCE_DATA_TAG -ErrorMessage "Unable to lookup VM tag `"$TagName`" from IMDS instance data"
+    }
+
+    $tag = $response.compute.tagsList | Where-Object { $_.name -eq $TagName }
+    if (!$tag) {
+        return $DefaultValue
+    }
+    return $tag.value
+}
+
+# Note: this function modifies global kubelet config args and node labels. Thus, it MUST
+# be called before Write-KubeClusterConfig, and any other function that relies on the values of
+# kubelet config args and node labels.
+function Disable-KubeletServingCertificateRotationForTags {
+    Logs-To-Event -TaskName "AKS.WindowsCSE.DisableKubeletServingCertificateRotationForTags" -TaskMessage "Check whether to disable kubelet serving certificate rotation via nodepool tags."

```

Comment:
It is better to add `$global:EnableKubeletServingCertificateRotation` in the log event.